.github/workflows/test.yml

```diff
@@ -328,17 +328,58 @@ jobs:
   test-link-verifier:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+
       - name: Setup python environment
-        uses: actions/setup-python@v2
+        uses: actions/setup-python@v3
+
+      - env:
+          stepName: "Functional | Success | Link Verifier Works & Excludes Directory"
+        name: ${{ env.stepName }}
+        id: link-verifier-success-case
+        uses: ./link-verifier
         with:
-          python-version: '3.11.0'
-      - name: Test link verifier action
+          path: ./
+          exclude-dirs: complexity, formatting, badFiles
+          allowlist-file: link-verifier/badUrls.md
+
+      - env:
+          stepName: "Functional | Success | Link being Excluded"
+        name: ${{ env.stepName }}
+        id: link-verifier-exclude-url-success-case
+        uses: ./link-verifier
+        with:
+          path: ./
+          exclude-dirs: complexity, formatting
+          exclude-urls: https://dummy-url.com/ota.bin, https://dummy-url.com/ota.com, https://dummy-url-three.com/ota.bin, https://github.com/FreeRTOS/A-Repo-That-Wins-You-The-Lottery
+
+      - env:
+          stepName: "Functional | Failure | Link Does Not Exist"
```

Comment:
Is this test not just a subset of `Functional | Success | Link Verifier Works & Excludes Directory`? Looks like both are checking the same `badUrls.md` file